credentialMixin.js

```diff
@@ -79,31 +81,53 @@ export const credentialMixin = {
       return this.issuer.name;
     },
     credentialImage() {
+      if(!this.credential) {
+        return '';
+      }
       if(this.credential.image) {
         return this.credential.image;
       }
-      if(this.issuer.image) {
-        return this.issuer.image;
-      }
-      if(this.issuer.logo) {
-        return this.issuer.logo;
+      const {credentialSubject = {}} = this.credential;
+      if(credentialSubject.image) {
+        return credentialSubject.image;
```

Comment:
This seems a bit questionable as it's a field of the subject vs of the credential itself.

Reply:
Yes, best if we don't conflate these.

Reply:
this was reverted to the original code in master. Let me know if this is correct:

master:
https://github.com/digitalbazaar/bedrock-vue-credential-card/blob/c86cc0b21b8ecae6ae9d6d146520d01db33c032e/credentialMixin.js#L88-L99

this branch:
https://github.com/digitalbazaar/bedrock-vue-credential-card/blob/03858a98e6e7ed76a74776df783b45b5c60f01f8/credentialMixin.js#L83-L94